website/docs/flow/index.md

```diff
@@ -1,80 +1,100 @@
 ---
-title: Overview
+title: Flows
 ---
 
-Flows are a method of describing a sequence of stages. A stage represents a single verification or logic step. They are used to authenticate users, enroll them, and more.
+Flows are a major component in authentik. In conjunction with stages and [policies](../policies/index.md), flows are at the heart of our system of building blocks, used to define and execute the workflows of authentication.
+
+There are over a dozen default, out-of-the box flows available in authentik. Users can decide if they already have everything they need with the default flows or if they want to [create](#create-a-custom-flow) their own custom flow, using the Admin interface.
+
+A flow is a method of describing a sequence of stages. A stage represents a single verification or logic step. By connecting a series of stages within a flow (and optionally attaching policies as needed) you can build a highly flexible process for authenticating users, enrolling them, and more.
 
 For example, a standard login flow would consist of the following stages:
 
--   Identification, user identifies themselves via a username or email address
--   Password, the user's password is checked against the hash in the database
--   Log the user in
+-   **Identification stage**: user identifies themselves via a username or email address
+-   **Password stage**: the user's password is checked against the hash in the database
```

Comment:
This list is removing the login part from above which is specific stage and should be included as such

Reply:
Wrong order, the login stage is last in the flow